HKMP/Api/Client/IClientManager.cs

```diff
@@ -90,4 +90,14 @@ public interface IClientManager {
     /// Event that is called when another player leaves the local scene.
     /// </summary>
     event Action<IClientPlayer> PlayerLeaveSceneEvent;
+
+    /// <summary>
+    /// Event that is called when HKMP modifies the game's time scale.
```

Comment:
This doc comment does not describe the delegate, but is a copy of the description of the event below it.